src/TextArea/index.tsx

```diff
@@ -2,25 +2,48 @@ import React from 'react';
 import getClassName from 'getclassname';
 
 export interface TextAreaProps {
-  name: string;
-  variant?: string;
+
+  id?: string;
+  isDisabled:boolean;
+  name?: string;
   hasIcon?: boolean;
-  textAreaProps?: React.ComponentPropsWithoutRef<"div">;
 }
 
-const TextArea = ({ name = "normal", variant = "outline", hasIcon = false, textAreaProps = {} }: TextAreaProps) => {
+const Padded = ({ children }) => <div style={{margin: "16px"}}> {children} </div>;
```

Comment:
Should this style be part of @e-toast/css?

Reply:
🤔 Good question I define this style following the [guides](https://github.com/bod-endava/e-toast-ds/blob/5f24b99af78ce8b6617908eecca214fc4db7b19f/packages/react-sandbox/src/App.js#L4-L6)that we have in the react-sandbox package that are at the beginning of the `App.js` file.

Reply:
After check, we can delete this style. Changes updated.